### PR TITLE
[TASK] Use templates to style admonitions

### PR DESCRIPTION
**Pull request by linawolf**
references https://github.com/doctrine/rst-parser/issues/6

### CODE REVIEW

**Comment by linawolf** [comment]
@greg0ire could you review please? I would like to merge this step before adding more admonitions

File: lib/Directives/Admonition.php [hunk 1]
```diff
@@ -5,31 +5,24 @@
...
+    private $text;
 
-    /** @var string */
-    private $textColor;
-
-    /** @var string */
-    private $icon;
-
-    public function __construct(string $name, string $backgroundColor, string $textColor, string $icon)
+    public function __construct(string $name, string $text)
```
Comment (greg0ire) [inline lib/Directives/Admonition.php:22]: This is a BC break, should this target 0.6.x?

**Comment by linawolf** [comment]
@greg0ire what else is missing?

**Review by greg0ire — APPROVED** [review]

**Review by SenseException — APPROVED** [review]

**Comment by greg0ire** [comment]
@linawolf nothing I think :) Thanks!

**Pull request merged**